utilities/ovs-vsctl.c

```diff
@@ -442,14 +448,40 @@ SSL commands:\n\
 Auto Attach commands:\n\
   add-aa-mapping BRIDGE I-SID VLAN   add Auto Attach mapping to BRIDGE\n\
   del-aa-mapping BRIDGE I-SID VLAN   delete Auto Attach mapping VLAN from BRIDGE\n\
-  get-aa-mapping BRIDGE              get Auto Attach mappings from BRIDGE\n\
-\n\
+  get-aa-mapping BRIDGE              get Auto Attach mappings from BRIDGE\n");
+}
+
+#ifdef P4OVS
```

Comment:
Will this be defined elsewhere in another commit/PR?

Reply:
The p4 commands were added in the previous commit.

Based on the conversation at today's meeting, the p4 commands may be backed out. (In fact, it sounds as if a large part of the first commit may be backed out.) The p4 bridge is an implementation mechanism for multiple pipelines, which we do not yet support.

Reply:
OK. Will wait for further commits to remove this code then if we are backing out on some p4 specific changes related to multiple pipelines